input_tool.py

```diff
@@ -0,0 +1,203 @@
+import pickle
+import json
+import os
+import time
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# can be modified to the file path of experiment data
+EXPERIMENT_DATA_PATH = os.path.join(os.getcwd(),'local')
+
+# environment variables for connecting to database
+db_host = os.environ.get('DB_HOST')
+db_port = os.environ.get('DB_PORT')
+db_name = os.environ.get('DB_NAME')
+db_user = os.environ.get('DB_USER')
+db_password = os.environ.get('DB_PASSWORD')
+
+def create_tables(cur):
+    # Users table
+    cur.execute("""CREATE TABLE IF NOT EXISTS owners (
+                owner_id serial PRIMARY KEY,
+                first_name VARCHAR,
+                last_name VARCHAR,
+                username VARCHAR NOT NULL UNIQUE
+    )""")
+    # Experiments table
+    cur.execute("""CREATE TABLE IF NOT EXISTS experiments (
+                experiment_id serial PRIMARY KEY,
+                experiment_name VARCHAR,
+                owner_id INTEGER NOT NULL,
+                CONSTRAINT fk_owner
+                    FOREIGN KEY (owner_id)
+                        REFERENCES owners(owner_id)
+                        ON DELETE CASCADE,
+                UNIQUE(experiment_name, owner_id)
+    )""")
+    # Groups table
+    cur.execute("""CREATE TABLE IF NOT EXISTS groups (
+                group_id serial PRIMARY KEY,
+                group_name VARCHAR
+    )""")
+    # Variables table
+    cur.execute("""CREATE TABLE IF NOT EXISTS variables (
+                variable_id serial PRIMARY KEY,
+                variable_name VARCHAR,
+                channel INTEGER
+    )""")
+    # Observations table
+    cur.execute("""CREATE TABLE IF NOT EXISTS observations (
+                observation_id serial PRIMARY KEY,
+                observation_name VARCHAR,
+                variable_id INTEGER NOT NULL,
+                CONSTRAINT fk_variable
+                    FOREIGN KEY (variable_id)
+                        REFERENCES variables(variable_id)
+                        ON DELETE CASCADE      
+    )""")
+    # Plots table
+    cur.execute("""CREATE TABLE IF NOT EXISTS plots (
```

Comment:
Do we need a `UNIQUE` in this table?

Reply:
Like a unique on `(experiment_id, group_id, and observation_id)` ?

Reply:
Yes, I can reflect this in the schema.